decidim-meetings/app/controllers/decidim/meetings/calendars_controller.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Meetings
+    # Exposes the meeting resources as an .ics file so users can import them
+    # to their favorite calendar app
+    class CalendarsController < Decidim::Meetings::ApplicationController
+      layout false
+      helper_method :meetings
+      before_action :set_default_request_format
+
+      def show
+        # TODO: add `content_type: "type/calendar"` before finishing
```

Comment:
Fix this

Reply:
I'm not sure it's needed. If we add this, the server will force the download of the file 😕

Reply:
Talked offline with @oriolgual and decided to force the download.